src/utils/zcl_abapgit_repo_labels.clas.abap

```diff
@@ -0,0 +1,245 @@
+CLASS zcl_abapgit_repo_labels DEFINITION
+  PUBLIC
+  FINAL
+  CREATE PUBLIC .
+
+  PUBLIC SECTION.
+
+    TYPES:
+      BEGIN OF ty_label_color,
+        label TYPE string,
+        color TYPE string,
+      END OF ty_label_color,
+      ty_label_colors TYPE STANDARD TABLE OF ty_label_color WITH KEY label.
+
+    CONSTANTS c_allowed_chars TYPE string VALUE `-_.a-zA-Z0-9` ##NO_TEXT.
```

Comment:
This won't work well for many languages. Any reason why there are not more chars allowed? Why not everything except `,` (and no control chars, of course)?

Reply:
I prefer to start with limited but surely working stuff. Disallowed set can be not that small. controls chars, html escaped chars, comma, semicolon ... there might be some other limitations. Also I haven't seen a lot of localized labels, I mean in other tools. 
Anyway I don't mind, it's just let's do it after the feature is stable and real feedbacks received.